src/shared/js/content.js

```diff
@@ -10,15 +10,38 @@ import { CSK_ENTRY_ID_NAME, CSK_ENTRY_SELECTOR } from './helpers/constants';
 import { resetBlur, setBlur } from './helpers/blur';
 import getContentfulItemUrl from './helpers/getContentfulItemUrl';
 
+const shrinkContent = () => {
```

Comment:
What is the purpose of the shrink/expand functions?

Reply:
When showing the sidebar  a `padding-left` is applied to the body to make room for it. If there elements with `position:fixed` those will be hidden behind the sidebar. This functions take care of those elements by adding a custom padding on each of them + whatever the original padding was.